### ISSUE TITLE
About the test set used for evaluation

### ISSUE THREAD

**Issue by tmliang** [Spico197/NYT-H#5]
Thank you very much for your work. As you said, the training set contains 50% NA instances, while the test set contains no NA instances. Then, reducing the category weight of NA (extremely, ignoring all NA instances) during training is undoubtedly an effective trick to make the results look better, but it is unfair and makes no sense. How can it be avoided?

**Comment by tmliang** [comment]
I think it might be more consistent if the test set is required to contain an NA set, just as train.json = train_nonna.json + na_train.json

**Comment by rainarch** [comment]
Since it is hard to ask human to label the instances in NA, we decided to remove the NA from the test set.

**Comment by Spico197** [comment]
Hi, there. Besides the reason of laboured annotation, NA instances are TNs under the DS assumption. Ideally, it should not affect the final results. However, these TNs are mixed with FNs due to the strong assumption. This kind of noise is a substantial evidence to the DS effects, so here we follow the conventional way to keep the NA instances in the train set.
